apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -195,6 +195,10 @@ function WpcomBlockEditorNavSidebar() {
 			>
 				<div className="wpcom-block-editor-nav-sidebar-nav-sidebar__header">
 					<Button
+						aria-label={ __(
+							'You are viewing the sidebar. To close select escape',
+							'full-site-editing'
+						) }
```

Comment:
I wonder a longer description like this (that includes instructions on how to use the sidebar) would be better as an `aria-description` attribute instead of a label. I haven't tried it on voiceover to see what it would actually do, but that seems to be how the [MDN article on ARIA annotations](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Annotations) describes the attribute.

Reply:
> 'You are viewing the sidebar. To close select escape'

Would something like this be more specific in terms of both the action (press) and the target of the action (the Esc key)?

'You are viewing the sidebar. Press the Escape key to close.'

Reply:
@p-jackson Your suggestion on the use of `aria-description` makes sense to me.  I'll make that change.

Also like your wording @ramonjd .  I wasn't sure how much leeway I had in changing the original design/issue suggestion as I also thought the phrase could be clearer. :) Thanks for the suggestion.

Reply:
Scratch that about the `aria-description`:
![Screen Shot 2020-09-01 at 12 50 10](https://user-images.githubusercontent.com/5665959/91789311-bf0c9380-ec51-11ea-8950-67acb542d5e8.png)

Reply:
I think we need to update the `eslint-plugin-jsx-a11y` plugin.

https://github.com/Automattic/wp-calypso/blob/master/package.json#L300

We're at 6.2.3, but `aria-description` is only support after 6.3: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/issues/699

Reply:
Or in the meantime, you could disable it, e.g., 

```

{ /* eslint-disable eslint-disable jsx-a11y/aria-props */ }
<Button
    aria-description={ ... }
    ...
/>
{ /* eslint-enable eslint-disable jsx-a11y/aria-props */ }

Then, once we upgrade remove the comments (?)